src/pages/blog.jsx

```diff
@@ -0,0 +1,56 @@
+import Image from 'next/image'
+
+import chevronDown from '../../public/icons/chevron-down.svg'
+import frequency1 from '../../public/images/frequency.png'
+import BlogItem from '../components/blog-item'
+import blogStyles from '../styles/blog.module.css'
+
+const blog = () => {
+  const data = [
+    {
+      image: '../../public/images/frequency.png',
```

Comment:
```suggestion
      image: frequency1,
```

I believe you can just pass the image directly (they don't seem to be displaying anyway on the site)